doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -909,16 +909,21 @@ hierarchical scheme for either reduction or top-k computation.
 
 ### `mz_message_counts`
 
-The `mz_message_counts` view describes the messages sent and received over the [dataflow] channels in the system.
+The `mz_message_counts` view describes the messages and message batches sent and received over the [dataflow] channels in the system.
+It distinguishes between individual records (`sent`, `received`) and batches of records (`batch_sent`, `batch_sent`).
```

Comment:
There is some potential for confusion here. We have:
* arrangement records (aka "updates") and arrangement batches in `mz_arrangement_sizes`
* message records and now message batches in `mz_message_counts`

IIUC these are not the same. To make matters worse, depending on the operator "message records" sometimes correspond to the number of updates transmitted, but other times they correspond to the number of arrangement batches (I think?).

I'm not sure how to resolve this confusion. Maybe we can come up with non-ambiguous names? E.g.:
* arrangement records -> updates
* arrangement batches -> batches
* message records -> records
* message batches -> messages

Reply:
I agree! What about using `containers` instead of `batches` for Timely-related logs? It's what's transferred in each message event, so it might come closer to Timely's terminology, as far as there is one.

Reply:
SGTM!

Reply:
Filed #21140 to capture this, thank you!